yarn.lock

```diff
@@ -13801,6 +13801,12 @@ __metadata:
   languageName: unknown
   linkType: soft
 
+"web-pocs@workspace:packages/web-pocs":
+  version: 0.0.0-use.local
+  resolution: "web-pocs@workspace:packages/web-pocs"
+  languageName: unknown
+  linkType: soft
+
```

Comment:
Excuse me, wir haben 2022?!

Where does the yarn.lock file come from? I don't see it in master and we shouldn't reads it